components.yaml

```diff
@@ -0,0 +1,19 @@
+componenets:
+  multus:
+    version: "v3.4"
```

Comment:
In this case, we don't use 3.4 directly, we use an untagged version that came after it.

You can see it in the image tag here: https://github.com/kubevirt/cluster-network-addons-operator/blob/master/pkg/components/components.go#L22

In this case, the version should be therefore set to `bf61002b4c83fcc0ca29c271108ca6afa381998f`

Reply:
is 
```version: "bf61002b4c83fcc0ca29c271108ca6afa381998f"```
 be good then? i.e., do you think future scripts will be able to interpret it in the same manner they do a specific version?

Reply:
We may need to add "branch" or "revision" later, but I think this is good enough for now. Let's adjust it once we have a real user and real issue.

I'm little nervous that we introduce this file and has some redundancy, but let's use it for reference and docs now, then we start basing automation (both D/S and U/S around it). We can change it whenever, it's no API, just our helper.